guides/plugins/apps/gateways/context/context-gateway.md

```diff
@@ -0,0 +1,305 @@
+# Context Gateway
+
+::: danger
+**Security and privacy**
+
+With the context gateway, Shopware allows your app to manipulate the customer context, which includes sensitive information like customer addresses, payment methods, and more.
+It is your responsibility to ensure that the commands are valid and do not compromise the security or privacy of customers.
+
+As this is a powerful feature, you should only use it if you are sure that your app server is secure and the commands you send are safe.
+
+:::
+
+## Context
+
+As of Shopware version 6.7.1.0, the Context Gateway was introduced.
+
+The Context Gateway is a powerful feature that allows apps to securely interact with the customer context, based on the current cart and sales channel and make informed decisions on the app server.
+The app system benefits from this solution, enabling app developers to manipulate customer contexts and provide a more tailored shopping experience.
+
+It is designed as to being the connection point between your apps Javascript and your app server.
+
+## Prerequisites
+
+You should be familiar with the concept of Apps, their registration flow as well as signing and verifying requests and responses between Shopware and the App backend server.
+
+<PageRef page="../../app-base-guide.md" title="App base guide" />
+
+Your app server must be also accessible for the Shopware server.
+You can use a tunneling service like [ngrok](https://ngrok.com/) for development.
+
+## Manifest configuration
+
+To indicate to Shopware that your app uses the context gateway, you must provide a `context` property inside a `gateways` parent property of your app's `manifest.xml`.
+
+Below, you can see an example definition of a working checkout gateway configuration.
+
+::: code-group
+
+```xml [manifest.xml]
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+    <!-- ... -->
+
+    <gateways>
+        <!-- ... -->
+        <context>https://my-app.server.com/context/gateway</context>
+    </gateways>
+</manifest>
+```
+
+:::
+
+After successful installation of your app, the context gateway is ready to be called by Shopware.
+
+## Context gateway endpoint
+
+Your JavaScript integration is responsible for calling the context gateway endpoint.
```

Comment:
I think, this needs clarification. I would not make this JS specific. I would describe, that an endpoint on the Shopware server needs to be called (either Store-API or Storefront), which then in turn calls the app server with the data that is passed. This ensures secured communication between Shopware and the App Server.